location/tests.py

```diff
@@ -1,3 +1,95 @@
-# from django.test import TestCase
+from django.db import IntegrityError
+from django.core.exceptions import ValidationError
+import pytest
+from .models import Location
 
-# Create your tests here.
+LOCATION_NAME = "location1"
+CITY = "city1"
+STREET = "street1"
+STREET_NUMBER = 1
+INDOOR = False
+DESCRIPTION = "description1"
+
+LOCATION_NAME_NEW = "location2"
+CITY_NEW = 'city2'
+STREET_NEW = 'street2'
+STREET_NUMBER_NEW = 2
+INDOOR_NEW = True
+DESCRIPTION_NEW = 'description2'
+
+LONG_FIELD = "a" * 45
+EMPTY_FIELD = ''
+SPECIAL_CHARACTERS = "special%@chars"
```

Comment:
you can also get all the special characters by doing:

```suggestion
import string

SPECIAL_CHARACTERS = string.punctuation
```
the output of string.punctuation is: '!"#$%&\'()*+,-./:;<=>?@[\\]^_`{|}~'